associations/views/library.py

```diff
@@ -37,13 +37,18 @@ def get_queryset(self):
 
         return Library.objects.filter(Q(enabled=True) | Q(id__in=libraries_id))
 
+    def get_serializer_class(self):
+        if self.action in ("create", "update", "partial_update"):
+            return LibraryWriteSerializer
+
+        return LibrarySerializer
+
 
 class LoanableViewSet(viewsets.ModelViewSet):
     queryset = Loanable.objects.all()
     serializer_class = LoanableSerializer
     permission_classes = (LoanablePermission,)
 
-    filter_backends = (DjangoFilterBackend,)
```

Comment:
Please keep this. It allows to make filtered queries from the rest API in the GET parameters